app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -56,23 +64,33 @@
     "Eritrea",
     "Estonia",
     "Ethiopia",
+    "Falkland Islands",
```

Comment:
ISO-3166-1 lists this as "Falkland Islands (Malvinas)" - I'm British, so refer to them as the Falkland Islands, but not having the "Malvinas" part on the end could upset any Argentinian users of the platform.